lib/utils/makeUseMdxComponents.ts

```diff
@@ -0,0 +1,34 @@
+import type { Depth } from '../../components/Heading';
+import type { UseMdxComponents } from '@mdx-js/mdx';
+import type { MDXComponents } from 'mdx/types';
+
+import Variable from '@readme/variable';
+
+import * as Components from '../../components';
+
+const makeUseMDXComponents = (more: ReturnType<UseMdxComponents> = {}): UseMdxComponents => {
```

Comment:
I had moved this to it's own file, cause I thought I could put the logic here. That wasn't true, but I still think it's nicer in it's own file.